rego/Groups.rego

```diff
@@ -357,8 +357,8 @@ tests contains {
     "NoSuchEvent": true
 }
 if {
-    DefaultSafe := false
-    Events := utils.FilterEventsOU(LogEvents, "GroupsSharingSettingsProto allow_unlisted_groups", utils.TopLevelOU)
+    DefaultSafe := true
+    Events := utils.FilterEvents(LogEvents, "GroupsSharingSettingsProto allow_unlisted_groups", utils.TopLevelOU)
```

Comment:
Same as above, why the `FilterEvents` function?

Reply:
reverted